### ISSUE TITLE
Volume up/down by Remote's Up/down button

### ISSUE THREAD

**Issue by rahaaatul** [nova-video-player/aos-AVP#510]
Hello. Can you add an option to enable volume up/down by the tv remotes up/down button or the keyboards up/down keys? Even better if we can remap buttons in-app.

Reason: My TV doesn't support Bluetooth. So I bought a Bluetooth transmitter/receiver to connect with my Bluetooth speaker. But the problem is, with the tv remote's volume button, doesn't change Bluetooth volume, it changes only tv's volume. So I need the in-app volume slider to change by remotes up/down button like MX Player.

**Comment by Pentaphon** [comment]
That seems quite unnecessary and out of the scope of this project.

**Comment by rahaaatul** [comment]
> That seems quite unnecessary and out of the scope of this project.

I don't know what's you're reason to think that, I just explained my reason to open this issue. If it was unnecessary, I wouldn't open this issue.

Anyway, I really need this option to be added because- 

> I have to close Nova player and go to other app to increase/decrease volume, and go back to Nova back again (using remote).

Please, help this option to be added by the developer. Thank you.

**Comment by courville** [comment]
@rahaaatul in the player volume can be increased via real keyboard with U/D (old mplayer shortcuts?) https://github.com/nova-video-player/aos-AVP/wiki/Keyboard-shortcuts
Sorry but I do not get really the setup: how you are controlling from the tv (it should be android handling that) the volume of your BT TX/RX?

**Comment by rahaaatul** [comment]
> @rahaaatul in the player volume can be increased via real keyboard with U/D (old mplayer shortcuts?) https://github.com/nova-video-player/aos-AVP/wiki/Keyboard-shortcuts

> Sorry but I do not get really the setup: how you are controlling from the tv (it should be android handling that) the volume of your BT TX/RX?

I thought so too, but it doesn't control the way I thought it would. 

In MX player, there's a Controls option called "Key up/down action", I enabled it in there and it changes the volume, but it changes in-app volume, but not the real volume (meaning: it thinks the originally set Bluetooth volume as the maximum volume, so if it's set to 50, MX thinks that's as 100). But Nova changes and is linked to the real volume, and I can change sound via mouse (wired) but not via remote's arrow buttons (which I'm requesting for an option to be added like up/down arrow changes volume and menu button brings the menu) . In Nova, the Up arrow buttons, bring the player menu. This is my [TV remote's layout](https://images.app.goo.gl/j6W3Sn4KM9KFXU9x7) (almost). This is the exactly same [USB](https://www.amazon.com/Bluetooth-Transmitter-Receiver-Portable-Wireless/dp/B07MFDD28G) I used for the Bluetooth receiver (notice that it's not plugged into the headphone jack, plugging into the headphone jack makes buzz sound, so it purely transmits sound into Bluetooth speaker wirelessly). This is the [AIDA64 report](https://drive.google.com/file/d/1QbJDuJ2ajcChnsisLqTrYU6lJ98enUxU/view?usp=drivesdk) of my tv.
